packages/rehype/types/index.d.ts

```diff
@@ -0,0 +1,13 @@
+// TypeScript Version: 3.5
+
+import {Processor} from 'unified'
+import {RehypeParserOptions} from 'rehype-parse'
+import {RehypeStringifyOptions} from 'rehype-stringify'
```

Comment:
`RehypeStringifyOptions` is different from `RehypeParserOptions` in name. Will this cause typos? Is `RehypeParseOptions` (no `r`) better?

Reply:
I agree using `RehypeParseOptions` is better for consistency.